code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -66,7 +66,7 @@
 	key_third_person = "screams"
 	message = "screams!"
 	emote_type = EMOTE_AUDIBLE
-	only_forced_audio = TRUE
+	cooldown = 10 SECONDS
```

Comment:
Just pointing this out for the fucktards that will downvote the pr because "you can spam it"